### ISSUE TITLE
Does it work on windows OS?

### ISSUE THREAD

**Issue by hina1929** [wagiminator/CH32V003-MacroPad-plus#1]
I created this ch32bv003-macropad plus using publicly available data but it does not work well on windowsOS.
 It works fine on chromeOS.
 Does this not work on windows OS? Or do I need to do something?
Translated by Deepl

**Comment by wagiminator** [comment]
What do you mean by "not well"? It should normally work with any operating system since HID is a standard. However, I don't have a Windows PC myself, so unfortunately, I can't help you further at the moment.

**Comment by hina1929** [comment]
Thanks for the reply.
 The rotary encoder only responds once every few times.
 As for the keyswitches, they are not responsive at all.
 I tried the media keys, modifier keys, and regular letters, but none of them responded.
 I thought it might be a slow process, so I implemented the "Mouse Wiggler" from your repository on a universal board and tried it on the windows PC in question, and it worked fine.
